welcome_wizard/tests/test_views.py

```diff
@@ -1,39 +1,156 @@
 """Tests for Welcome Wizard Views."""
 
 from django.contrib.auth import get_user_model
-from django.test import override_settings
+from django.test import Client, override_settings
+from django.utils.text import slugify
 from django.urls import reverse
 
+from nautobot.extras.models import Tag
+from nautobot.users.models import ObjectPermission
+from nautobot.utilities.permissions import resolve_permission_ct
+from nautobot.utilities.testing.utils import extract_form_failures
+
 from nautobot.dcim.models import Manufacturer, DeviceType
 from nautobot.dcim.models.sites import Site
+from nautobot.utilities.testing import TransactionTestCase
 from nautobot.utilities.testing.views import TestCase
 
 from welcome_wizard.models.importer import DeviceTypeImport, ManufacturerImport
-from welcome_wizard.tests.test_datasources import nautobot_jobs_logs_decorator
+
 
 User = get_user_model()
 
 
-@nautobot_jobs_logs_decorator()
-class ManufacturerTestCase(TestCase):
+class WizardTestCaseMixin:
+    """Mixin for WelcomeWizard Tests. This is temporary until tests move to running against Nautobot 1.4"""
+
+    user_permissions = ()
+
+    def setUpWizard(self, client=True):  # pylint: disable=invalid-name
+        """Setup shared testuser, statuses and client."""
+
+        # Create the test user and assign permissions
+        self.user = User.objects.create_user(username="testuser")
```

Comment:
Isn't this exactly the kind of usage that https://github.com/nautobot/nautobot/pull/1954 would potentially break? I.e. if TransactionTestCase starts automatically creating `testuser` for you, then this call here would start failing since the requested user already exists.

Reply:
Yep, I changed this to use a different user, but was troubleshooting permissions and forgot to change back, I will correct this.